ocs_ci/templates/workloads/fio/fedora_deployment.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fio_deployment_raw_block
+  labels:
+    app: fio-benchmark
+    name: fio_deployment_raw_block
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fio_deployment_raw_block
+  template:
+    metadata:
+      labels:
+        app: fio_deployment_raw_block
+    spec:
+      containers:
+        - name: fio
+          image: datawiseio/fio:latest
```

Comment:
We might need to look for image that's available in quay as we might get rate limit also the image is 8 year old

Reply:
To be tracked with https://github.com/red-hat-storage/ocs-ci/issues/8087